src/comp/comp.py

```diff
@@ -24,48 +24,49 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [human for human in humans if human.name[0] == "D"]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [human for human in humans if human.name[-1] == "e"]
```

Comment:
Same case as "starts with D". Output is different from what was required by tests.